Program.cs

```diff
@@ -1,22 +1,14 @@
-﻿namespace test1
+﻿using System.Collections.Generic;
+using System;
+
+namespace test1
 {
     public class Program
     {
         static void Main(string[] args)
         {
-            BaseDataContacts newBook = new("test1");
-            newBook.InitializationBD();
-            RenderContact render = new(newBook, 4);
-            MenuInput input = new(render, newBook);
-
-            int page = 1; //какую страницу рендерить дальше
-            do
-            {
-                render.RenderPage(page);
-                page = input.MainInput();
-            }
-            while (page != 0);
+            Screen screenChoise = new ScreenChoise(3);
```

Comment:
```suggestion
            Screen screenChoiсe = new ScreenChoiсe(3);
```
https://translate.google.com/?sl=en&tl=ru&text=choice&op=translate